subtitles/ZAP Chat/README.md

```diff
@@ -0,0 +1,35 @@
+# Generate subtitles for ZAP Chat using whisper
+
+Here is a short guide on how the subtitles were created using OpenAI's Whisper.
+A <b>Nvidia graphics card with Cuda cores is required</b>. The more VRAM the better, as larger models can then be used if necessary.
+In addition to Whsiper, you also need an audio file, e.g. in .mp3 format, which you want to transcribe.
+
+
+
+## Install Whisper
+
+First you have to install Whisper - you can find out how to do this in the official repository <a href="https://github.com/openai/whisper">https://github.com/openai/whisper.</a> Here you will also find a list of the requirements, languages and possibilities of what you can do with it.
```

Comment:
These should be MD links, ie `[text](url)`

Reply:
done